pxr/imaging/rprUsd/schema.usda

```diff
@@ -0,0 +1,595 @@
+#usda 1.0
+(
+    """ This file contains rpr usd schema to allow
+        a complete render of a rpr scene within HdRpr.
+    """
+    subLayers = [
+        @usd/schema.usda@
+    ]
+)
+
+over "GLOBAL" (
+    customData = {
+        string libraryName       = "rprUsd"
+        string libraryPath       = "."
+        string libraryPrefix     = "RprUsd"
+        dictionary libraryTokens = {
+            dictionary rpr = {
+                string doc = """."""
+            }
+            dictionary rprMinValue = {
+                string doc = """RPR specific Sdf metadata"""
+            }
+            dictionary rprMaxValue = {
+                string doc = """RPR specific Sdf metadata"""
+            }
+            dictionary rprHidden = {
+                string doc = """RPR specific Sdf metadata"""
+            }
+        }
+    }
+) {
+}
+
+class "RprRendererSettingsAPI" (
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "RendererSettingsAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaAutoApplyTo = ["RenderSettings"]
+    }
+    doc = "Defines render settings for AMD Radeon Pro Renderer"
+)
+{
+    uniform token rpr:core:renderQuality = "Northstar" (
+        allowedTokens = ["Low", "Medium", "High", "HybridPro", "Full", "Northstar"]
```

Comment:
We need to come up with a better name than HybridPro for this

Full mode should be -> Northstar
Legacy -> Tahoe